lib/services/local/trust.go

```diff
@@ -80,7 +80,7 @@ func (s *CA) UpsertCertAuthority(ca services.CertAuthority) error {
 // if the existing value matches existing parameter, returns nil if succeeds,
 // trace.CompareFailed otherwise.
 func (s *CA) CompareAndSwapCertAuthority(new, existing services.CertAuthority) error {
-	if err := new.Check(); err != nil {
+	if err := new.CheckAndSetDefaults(); err != nil {
```

Comment:
Shouldn't this be `ValidateCertAuthority`? `Check` was doing more than basic validation here right?

Same question elsewhere this is done.